beacon_node/network/Cargo.toml

```diff
@@ -48,4 +48,8 @@ operation_pool =  { path = "../operation_pool" }
 execution_layer =  { path = "../execution_layer" }
 beacon_processor =  { path = "../beacon_processor" }
 parking_lot = "0.12.0"
-environment = { path = "../../lighthouse/environment" }
\ No newline at end of file
+environment = { path = "../../lighthouse/environment" }
+
+[features]
+# default = ["no-backfill"]
+no-backfill = []
```

Comment:
the name you used below is `disable_backfill`

Reply:
:sweat_smile: